interfaces/cython/cantera/composite.py

```diff
@@ -637,22 +647,68 @@ def collect_data(self, cols=('extra','T','density','Y'), threshold=0,
 
         return np.hstack(data), labels
 
-    def write_csv(self, filename, cols=('extra','T','density','Y'),
+    def write_csv(self, file_name, cols=('extra','T','density','Y'),
```

Comment:
I think this should remain `filename` to avoid problems with anyone who might be using the keyword arguments here. Filename can be considered one word 😄 Same applies to `write_hdf()`.

Reply:
NP. had pep8 in mind and it wasn't a keyword. But will change as it follows precedent.

Reply:
A user can still use the argument as a keyword argument, even if there's no default value provided, i.e., this will work:

```python
states.write_csv(filename='states.csv')
```

The argument is mandatory, that's all.

Reply:
ah. for some reason I have never used that